src/fuzz_introspector/datatypes/function_profile.py

```diff
@@ -64,6 +64,12 @@ def __init__(self, elem: Dict[Any, Any]) -> None:
         # Saving callsites for this function
         try:
             self.callsite = self.load_func_callsites(elem['Callsites'])
+            # Add missed function called for this function
+            for func_name in self.callsite.keys():
+                if func_name not in self.functions_called:
+                    self.functions_called.append(func_name)
+                if func_name not in self.functions_reached:
+                    self.functions_reached.append(func_name)
```

Comment:
This extends the functions reached by a given fuzzer, which makes sense. However, we should keep it in mind considering it may have impact beyond the sink analyser.